### ISSUE TITLE
no reaction after like one day. restart fixes it for a while

### ISSUE THREAD

**Issue by mrcsmxms** [marcelkordek/homebridge-tesvor#2]
**Describe The Bug:**

when you install the plugin and edit your login-data into the configuration everything works just fine. but after a while (maybe like one day) the robot wont react to the switch in HomeKit. in homebridge there is no output from this plugin

**To Reproduce:**

everytime like one day after boot/reboot of homebridge

**Expected behavior:**

constant functionality

**Logs:**

```
[14/07/2021, 11:24:42] Loaded plugin: homebridge-tesvor@1.0.6-beta.4
[14/07/2021, 11:24:42] Registering platform 'homebridge-tesvor.HomebridgeTesvor'
[14/07/2021, 11:24:45] [HomebridgeTesvor] Initializing HomebridgeTesvor platform...
[14/07/2021, 11:24:46] [HomebridgeTesvor] Loading accessory from cache: vacuum
[14/07/2021, 11:24:52] [HomebridgeTesvor] Restoring existing accessory from cache: vacuum
```

**Plugin Config:**

```json
{
    "username": "XXX@X.de",
    "password": "XXX",
    "country": "+49",
    "startMode": "AutoClean",
    "stopMode": "BackCharging",
    "platform": "HomebridgeTesvor"
},
```

**Environment:**

* **Plugin Version**: homebridge-tesvor v1.0.6-beta.4 (also tried it in stable version, same issue)
* **Homebridge Version**: v1.3.4
* **Node.js Version**: v14.15.1
* **NPM Version**: v6.14.10
* **Operating System**:  Raspbian GNU/Linux Buster (10)
--

**Comment by marcelkordek** [comment]
Hi Marcus, thanks for the info. 
I‘ll have a look at it.

greetings
Marcel

**Comment by marcelkordek** [comment]
Should be fixed with 1.0.7

**Comment by jeankone** [comment]
Hi Marcel,

Still have the same problem on 1.0.7 :(
